test/hw3/test_task4.py

```diff
@@ -0,0 +1,19 @@
+from All_home_works.hw3.task4 import is_armstrong
+
+
+def test_is_armstrong_case1():
```

Comment:
`case1` does not mean anything to me.  Please give a more meaningful name for the test.